_events/2023-11-10-fosdem-cfp-2024.md

```diff
@@ -0,0 +1,89 @@
+
+Here are some topics that may inspire your talk:
+
+- Your experiences contributing to design in an open source project
+
+- How you use FLOSS tools to design
+
+- An open source project which is an example of great design
+
+- Your design process, research methods, and go-to resources while working with open source projects
+
+- Your ideas on fostering diversity and inclusion in design and open source
+
+- Anything else related to design and FLOSS!
+
+### Some examples of previous talks featured at the Open Source Design devroom at FOSDEM are:
+
+[Building a UX research toolkit: How a UX Research Toolkit is being built for the Open Source Ecosystem](https://archive.fosdem.org/2023/schedule/event/building_a_ux_research_toolkit/) by Mogashni Naidoo at FOSDEM 2023
+
+[Accessibility & Open Source: How open source is key to building a more inclusive world](https://archive.fosdem.org/2023/schedule/event/accessibility_and_open_source/) by Mike Gifford at FOSDEM 2023
+
+[Calling all UX Designers! Attracting and keeping UX designers on FOSS projects](https://archive.fosdem.org/2022/schedule/event/osd_attracting_and_keeping_ux_designers_on_foss_projects/) by Scott Jenson at FOSDEM 2022
+
+[Documenting a Design System: Lessons learned from open sourcing the Orbit docs](https://archive.fosdem.org/2022/schedule/event/osd_lessons_learned_from_open_sourcing_the_orbit_docs/) by Aaron Collier at FOSDEM 2022
+
+[Mentoring designers in open Source Software: What I've learned mentoring and supporting designers in OSS](https://archive.fosdem.org/2021/schedule/event/mentoring_designers_in_open_source_software/) by Eriol Fox at FOSDEM 2021
+
+[Open Source Design movement in Africa](https://archive.fosdem.org/2020/schedule/event/open_source_design_africa/) by Peace Ojemeh at FOSDEM 2020
+
+## What is the talk format?
+
+The majority of talks will be in person, and we expect some to be remote. The length of your talk should be **20 minutes** with **5 mins** for questions, or **25 minutes** without questions.
+
+- If you are giving your talk in person, you will deliver the talk live and you need to be physically present at the event in Brussels.
+
+- If you are giving your talk remotely, you need to send in a pre-recorded talk and you are required to be online with a stable internet connection when it is being presented and for 5 to 10 minutes after it is done for questions.
+
+## How can I submit a talk?
+
+This year all talks will be in person, with each slot for a talk lasting 25 minutes with 5 mins for questions from the audience. Talks cannot be longer than 30 mins otherwise you get gently moved off 'stage'. 
```

Comment:
On the duration, we really need those 5 minutes to allow people to enter and leave the room. So the talks should not be longer than 25 minutes (rather than 30).